README.md

```diff
@@ -1 +1,18 @@
-# hw01
\ No newline at end of file
+# hw01
+ 
+## **All about Whitney B**
+![alt text](https://github.com/whitneybeamer/hw01/blob/master/IMG_4622-1.JPG)
+
+*Hi! I am Whitney Beamer and here are a few facts about me:*  
+
+*I am a fourth year.  
+*I am a Public Policy and Comparative Human Development major.  
+*I am from Orlando, Florida.  
+*I am a huge Chance the Rapper fan!  
```

Comment:
Make sure to check the rendered version of the document to make sure everything is formatted correctly. For lists, you need a space between the `*` and the content

```
* I am a fourth year.  
* I am a Public Policy and Comparative Human Development major.  
* I am from Orlando, Florida.  
* I am a huge Chance the Rapper fan!  
```

* I am a fourth year.  
* I am a Public Policy and Comparative Human Development major.  
* I am from Orlando, Florida.  
* I am a huge Chance the Rapper fan!